scene/3d/vehicle_body_3d.cpp

```diff
@@ -413,9 +418,7 @@ real_t VehicleBody3D::_ray_cast(int p_idx, PhysicsDirectBodyState3D *s) {
 	ray_params.exclude = exclude;
 	ray_params.collision_mask = get_collision_mask();
 
-	bool col = ss->intersect_ray(ray_params, rr);
-
-	wheel.m_raycastInfo.m_groundObject = nullptr;
+	bool col = ss->intersect_ray(ray_params, rr);	
```

Comment:
The logic here shouldn't be changed. It's safer to first set groundObject to nullptr. Otherwise there's a potential case somewhere below where the raycast returns an invalid collider and the ground object would stay set to the previous one instead of being null.